app/controllers/prison/visits_controller.rb

```diff
@@ -9,18 +9,26 @@ def process_visit
     @visit = load_visit.decorate
   end
 
+  # rubocop:disable Metrics/MethodLength
```

Comment:
Yes, it is ugly, suggestions welcome. This was something I wanted to get ideas on.